frontend/lib/src/components/graph.dart

```diff
@@ -19,138 +18,213 @@ class Graph extends StatelessWidget {
   Graph({
     super.key,
     this.onTap,
+    required this.accountId,
   });
 
   final GraphController graphController = Get.put(GraphController());
   final SpendingBreakdownController spendingBreakdownController =
       Get.put(SpendingBreakdownController());
 
   final void Function()? onTap;
-
+  final String accountId;
```

Comment:
this is unnecessary, account id is already available as screen args

![image](https://github.com/framgia/sph-flutter/assets/95029803/23138a48-2daa-4192-b9f3-aa77a3eb92f4)